.github/workflows/mypy-validation.yml

```diff
@@ -20,10 +20,15 @@ on:
     paths:
       - 'src/controller/python/matter/ChipDeviceCtrl.py'
       - 'src/python_testing/matter_testing_infrastructure/**/*.py'
+      - 'scripts/tests/chipyaml/paths_finder.py'
+      - 'scripts/tests/chiptest/**/*.py'
```

Comment:
Why 'scripts/tests/chiptest/**/*.py' is not in the pull_request>paths ?

Reply:
Good catch